modular_skyrat/modules/container_emotes/code/container_emotes.dm

```diff
@@ -39,11 +41,11 @@
 			if("Audible")
 				emote_type = EMOTE_AUDIBLE
 			else
-				alert("Unable to use this emote, must be either audible or visible.")
+				user.balloon_alert(user, "Unable to use this emote, must be either audible or visible.")
```

Comment:
I wouldn't use a balloon alert, I meant to use the `tgui_alert()` or whatever it's actually called, I forgot. The thing that you just click "ok"